### PR TITLE
Refactor with Rector

### PR DESCRIPTION
**Pull request by provokateurin**
I had to rename the top-level files to add the .php extension, as otherwise rector refuses to touch them.

### CODE REVIEW

File: generate-spec.php
Comment (nickvergessen) [inline generate-spec.php:1]: Breaks existing CI?

Comment (provokateurin) [reply generate-spec.php:1]: Only by accident, nothing is actually broken.

**Comment by nickvergessen** [comment]
CI needs adjusting:
> /home/runner/actions-runner/_work/_temp/052b8c30-772d-47c1-920d-48b80d76ed3b.sh: line 3: ../generate-spec: No such file or directory

**Comment by provokateurin** [comment]
Which job? I already fixed that :thinking:

**Comment by nickvergessen** [comment]
> Which job? I already fixed that 🤔

"Generate OpenAPI - Server"

It has another call to generate-…

File: src/Helpers.php [hunk 1]
```diff
@@ -252,16 +252,16 @@ public static function getOpenAPIAttributeTagsByScope(ClassMethod|Class_|Node $n
 							if ($item?->value instanceof String_) {
 								$tag = $item->value->value;
 								$pattern = '/^[0-9a-zA-Z_-]+$/';
-								if (!preg_match($pattern, $tag)) {
+								if (in_array(preg_match($pattern, $tag), [0, false], true)) {
```
Comment (nickvergessen) [inline src/Helpers.php:255]: Do we mean  !== false?
or really also 0?

Comment (provokateurin) [reply src/Helpers.php:255]: False is failure (I guess invalid regex?) and 0 means no match, so this check is correct

**Pull request merged**